README.md

```diff
@@ -46,7 +46,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:5.8.0-SNAPSHOT'
+    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:6.0.0-SNAPSHOT'
```

Comment:
This also needs to be updated in the properties file.

Reply:
Ah nvm, this is dynamic in this project: https://github.com/mapbox/mapbox-java/blob/7c64c0464a55e2ed7ddcebc3c08edd962ab94424/gradle/artifact-settings.gradle#L35-L40

Reply:
it was removed from the properties. should I bring it back?